miden/src/cli/prove.rs

```diff
@@ -61,15 +61,9 @@ impl ProveCmd {
     }
 
     pub fn execute(&self) -> Result<(), String> {
-        println!("============================================================");
-        println!("Prove program");
-        println!("============================================================");
-
-        // configure logging
-        env_logger::Builder::new()
-            .format(|buf, record| writeln!(buf, "{}", record.args()))
-            .filter_level(log::LevelFilter::Debug)
-            .init();
+        log::info!("============================================================\n");
+        log::info!("Prove program\n");
+        log::info!("============================================================\n");
```

Comment:
After thinking a bit more, I'm wondering if things like this should remain `println!`. One can argue that this is not really logging output but an expected output of the command. That is, we should print out this and other statements in this file regardless of whether logging is enabled or not.